national-resources/norway.md

```diff
@@ -0,0 +1,176 @@
+* [NOVA – Novel approaches for design and evaluation of cost-effective surveillance across the food chain.](https://app.cristin.no/projects/show.jsf?id=624907) (2018-2020)
+* [MATRIX: Connecting dimensions in One-Health surveillance](https://app.cristin.no/projects/show.jsf?id=2560129) (2020-2022)
+* [NORSE - Network for One Health Resistome Surveillance](https://app.cristin.no/projects/show.jsf?id=2521986) (2020-2023)
+* [CORNELIA: AMR in One Health Interfaces](https://app.cristin.no/projects/show.jsf?id=2524169) (2021-2024)
+
+FHI maintains numerous projects related to infectious diseases both nationally and internationally:
+* [Avløpsovervåking SARS-CoV-2](https://www.fhi.no/cristin-prosjekter/aktiv/avlopsovervaking-sars-cov-2/) is a wastewater monitoring project with [regularly updated results](https://www.fhi.no/en/hn/statistics/overvaking-smittsomme-sykdommer-i-avlopsvann/results-from-wastewater-surveillance/) to better understand the spread of SARS-CoV-2 variants. [Project index on CRIStin](https://app.cristin.no/projects/show.jsf?id=2537724) (2022-2023)
+* The Norwegian influenza study (NorFlu) studies effects of influenza vaccination, illness, and treatment during pregnancy. [Project index on CRIStin](https://app.cristin.no/projects/show.jsf?id=560348) (2010-2025)
+* Corona-Reg studies the long-term health effects of Covid-19 and the Covid-19 pandemic on the Norwegian population. [Project index on CRIStin](https://app.cristin.no/projects/show.jsf?id=2058809) (2020-2025)
+* Covid-19 utbrudd i Norge- Epidemiologi, bruk av helsetjenester og håndtering i primærhelsetjenesten – CONOPRI investigates handling of the Covid-19 pandemic by primary health care services. [Project index on CRIStin](https://app.cristin.no/projects/show.jsf?id=2151826) (2020-2025)
+* Building Stronger Public Health Institutions and Systems (BIS) aims to strengthen public health in Norway and abroad. [Project index on CRIStin](https://app.cristin.no/projects/show.jsf?id=2527375) (2021-2025)
+* Consolidation of the HERA-WGS-infrastructure and capacity building at NIPH to enhance microbial surveillance and preparedness. [Project index on CRIStin](https://app.cristin.no/projects/show.jsf?id=2571637) (2022-2025)
+* Infeksjonssykdommers epidemiologi i Østfold analyses registry-based information on infectious diseases in Norway, especially Østfold. [Project index on CRIStin](https://app.cristin.no/projects/show.jsf?id=535246) (2016-2026)
+
+You will find information on other surveillance systems and databases under the sections [Dashboards and visualisation platforms](https://www.infectious-diseases-toolkit.org/national-resources/norway#disease-surveillance-initiatives) and [National data sources](https://www.infectious-diseases-toolkit.org/national-resources/norway#national-data-sources).
```

Comment:
The links in this piece are broken, would be better if they were more 'relative' links so e.g. /#navigating-the-dashboard/, because this would enable them to be tested prior to publication

Reply:
@bedroesb As similar to RDMkit where we have page ID https://rdmkit.elixir-europe.org/editorial_board_guide#related-pages, is it possible in IDTk to link to sections (within a page) with heading/ID? instead of full link to a specific section of the page Thanks!

Reply:
you should be able to do the markdown like: [Dashboards and visualisation platforms](#disease-surveillance-initiatives) and [National data sources](#national-data-sources).

Reply:
Thanks @LianeHughes updated the links, hopefully the links work fine on published page.